geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/Transformers.scala

```diff
@@ -117,16 +146,24 @@ object Transformers extends JavaTokenParsers {
   case class LitString(value: String) extends Lit[String]
   case class LitInt(value: Integer) extends Lit[Integer]
   case class LitLong(value: Long) extends Lit[Long]
+  case class LitFloat(value: java.lang.Float) extends Lit[java.lang.Float]
   case class LitDouble(value: java.lang.Double) extends Lit[java.lang.Double]
+  case class LitBoolean(value: java.lang.Boolean) extends Lit[java.lang.Boolean]
+
   case class Cast2Int(e: Expr) extends Expr {
     override def eval(args: Array[Any])(implicit ctx: EvaluationContext): Any = e.eval(args).asInstanceOf[String].toInt
   }
-
+  case class Cast2Long(e: Expr) extends Expr {
+    override def eval(args: Array[Any])(implicit ctx: EvaluationContext): Any = e.eval(args).asInstanceOf[String].toLong
```

Comment:
Maybe the easiest thing short term is in the conversion process, we could try/catch each individual attribute, instead of all attributes. That way if something like this fails it will still load the feature, just without that one attribute.